iree_tests/onnx/import_tests_utils.py

```diff
@@ -0,0 +1,46 @@
+import struct
+import numpy as np 
+
+# map numpy dtype -> (iree dtype, struct.pack format str)
+dtdict = {
+    np.dtype('int64'): ('si64', 'q') ,
+    np.dtype('uint64'): ('ui64', 'Q') ,
+    np.dtype('int32'): ('si32', 'i') ,
+    np.dtype('uint32'): ('ui32', 'I') ,
+    np.dtype('int16'): ('si16', 'h') ,
+    np.dtype('uint16'): ('ui16', 'H') ,
+    np.dtype('int8'): ('si8', 'b') ,
+    np.dtype('uint8'): ('ui8', 'B') ,
+    np.dtype('float64'): ('f64', 'd') ,
+    np.dtype('float32'): ('f32', 'f') ,
+    np.dtype('float16'): ('f16', 'e') ,
+    np.dtype('bool'): ('i1', '?') ,
+}
+
+def getShapeString(torchtensor):
+    inputshape = list(torchtensor.shape)
+    inputshapestring = "x".join([str(item) for item in inputshape])
```

Comment:
follow PEP 8 naming? https://peps.python.org/pep-0008/#function-and-variable-names

`def get_shape_string(torch_tensor)`, `input_shape = ...`, etc.

Should at least be consistent within the utils file.